src/events/message/messageCreate.ts

```diff
@@ -24,6 +25,31 @@ export default class extends Event {
 
 	public async run(message: Message): Promise<any> {
 		const context = await getCommandContext(message);
+		Sentry.setContext("user", {
+			tag: context.author.tag,
+			id: context.author.id, // @ts-ignore
+			channelPerms: context.channel.permissionsFor(context.author).bitfield || -1,
```

Comment:
Is there a reason this is `-1` for a default instead of `0` (which would mean no channel-specific permissions)?